sources/Adapters/picoTracker/gui/picoTrackerGUIWindowImp.cpp

```diff
@@ -8,6 +8,7 @@
 #include "Adapters/picoTracker/utils/utils.h"
 #include "Application/Utils/char.h"
 #include "UIFramework/BasicDatas/GUIEvent.h"
+#include "picoRemoteUI.h"
```

Comment:
I would just #ifdef this rather than the content of the header file

Reply:
ah right, good point!